src/components/Player.vue

```diff
@@ -629,25 +658,70 @@ li.move:not(.from) .player .overlay svg.move {
 
 /***** Player name *****/
 .player > .name {
-  text-align: center;
+  right: 10%;
+  display: flex;
+  justify-content: center;
   font-size: 120%;
   line-height: 120%;
   cursor: pointer;
   white-space: nowrap;
-  width: 100%;
+  width: 120%;
   background: rgba(0, 0, 0, 0.5);
   border: 3px solid black;
   border-radius: 10px;
   top: 5px;
   box-shadow: 0 0 5px black;
-  text-overflow: ellipsis;
-  overflow: hidden;
   padding: 0 4px;
 
+  svg {
+    top: 3px;
+    margin-right: 10px;
```

Comment:
```suggestion
    margin-left: 2px;
```
2px should be enough of a gap